lib/bowling.rb

```diff
@@ -0,0 +1,82 @@
+class Bowling 
+
+  def initialize
+    @frame = ""
+    @roll = ""
+    @pins = ""
+    @pins_array = []
+    @current_frame = []
+    @score_array = []
+    @spare = []
+    @strike = []
+  end 
+
+  # method with users input 
+  # def frames
+  #   puts "What frame are you currently playing?"
+  #   frame = gets.chomp.to_s
+  #   @frame = frame
+  #   binding.irb
+  # end
+
+  def frames(frame)
```

Comment:
I would suggest considering using initiailize to your advantage a bit more and passing it the frames, rolls and pins directly.